packages/cli/src/controllers/users.controller.ts

```diff
@@ -116,6 +120,15 @@ export class UsersController {
 			);
 		}
 
+		if (!isWithinUsersQuota) {
+			this.logger.debug(
+				'Request to send email invite(s) to user(s) failed because the user limit quota has been reached',
+			);
```

Comment:
Why are we logging the failure here but not in the others? Let's log in all or none?

Reply:
Added logging everywhere